tests/mocha/jso_deserialization_test.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+suite('JSO Deserialization', function() {
+  setup(function() {
+    sharedTestSetup.call(this);
+    this.workspace = new Blockly.Workspace();
+  });
+
+  teardown(function() {
+    workspaceTeardown.call(this, this.workspace);
+    sharedTestTeardown.call(this);
+  });
+
+  suite('Events', function() {
+    test.skip('Finished loading', function() {
```

Comment:
Will this be unskipped in an upcoming PR?

Reply:
Yesh this gets unskipped in the next one.